learning/tour-of-beam/frontend/pubspec.yaml

```diff
@@ -38,17 +38,21 @@ dependencies:
   flutter_svg: ^1.0.3
   get_it: ^7.2.0
   google_fonts: ^3.0.1
-  playground_components: { path: ../../../playground/frontend/playground_components }
+  json_annotation: ^4.7.0
+  playground_components:
+    { path: ../../../playground/frontend/playground_components }
```

Comment:
Unbreak this please. This will not build.

Reply:
My formatter does this automatically. `playground_components` seems to work fine. How can I see it fail to build?

Reply:
I'm surprised this works.
Please do not format this file with your formatter then. Or raise the line length limit.
The whole point of using `{ }` in YAML is to avoid line breaks so the list is uniform and is easier to read.